docker-compose.yml

```diff
@@ -18,7 +18,7 @@ services:
             - mongo
             - redis
         volumes:
-            - ~/sharelatex_data:/var/lib/sharelatex
+            - /var/sharelatex_data:/var/lib/sharelatex
```

Comment:
My only objection to changing this is that it'll conflict with existing deployments (where folks are using these repositories directly). 

Can we live without this change?

Reply:
Yes we can. I wanted to make life easier to ServerPro joiners, but I also agree not breaking existing stuff has higher priority.